docs/ref/databases.txt

```diff
@@ -941,6 +941,27 @@ To enable the JSON1 extension you can follow the instruction on
 .. _JSON1 extension: https://www.sqlite.org/json1.html
 .. _the wiki page: https://code.djangoproject.com/wiki/JSON1Extension
 
+.. _sqlite-init-command:
+
+Setting PRAGMA Options
+----------------------
+
+.. versionadded:: 5.1
+
+Pragma options can be set upon connection by using the ``init_command`` option
+similar to the MySQL backend. The example below shows how to disable
```

Comment:
`similar to the MySQL backend.` is unnecessary. There is not much description in the MySQL docs that could be helpful for SQLite users.

Reply:
Removed.